docker/settings.py

```diff
@@ -171,3 +171,14 @@
 SUPPORTTOOLS_PARENT_APP_URL = "/"
 
 REMOTE_USER_FORMAT = "uwnetid"
+
+MOCK_SAML_ATTRIBUTES = {
+    'uwnetid': ['javerage'],
+    'affiliations': ['student', 'member'],
+    'eppn': ['javerage@washington.edu'],
+    'scopedAffiliations': ['student@washington.edu', 'member@washington.edu'],
+    'isMemberOf': ['u_test_group', 'u_test_another_group'],
```

Comment:
It'd be better that this 'affiliations' mock data matches the mock PWS data
"EduPersonAffiliations":["member","student","alum","staff","employee"],

'isMemberOf':  ['u_astratst_myuw_test-support-admin']